math/vecops/CMakeLists.txt

```diff
@@ -33,13 +33,6 @@ elseif(vdt)
   target_link_libraries(ROOTVecOps PUBLIC ${VDT_LIBRARIES})
 endif()
 
-if(MSVC)
-  target_compile_options(ROOTVecOps PRIVATE -O2 /fp:fast)
-  target_compile_definitions(ROOTVecOps PRIVATE _USE_MATH_DEFINES)
```

Comment:
IIRC, `_USE_MATH_DEFINES` is independent of `/fp:fast` and should most probably be kept